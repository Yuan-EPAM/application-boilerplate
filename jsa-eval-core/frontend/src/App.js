import { hot } from 'react-hot-loader/root';
import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';

import SignUp from './components/SignUp';
import Dashboard from './components/Dashboard';

function App() {
  return (
    <Router>
      <Switch>
        <Route exact path="/">
          <div>
            <SignUp />
          </div>
        </Route>
        <Route path='/dashboard'>
          <Dashboard />
        </Route>
      </Switch>
    </Router>
  );
}

export default hot(App);
