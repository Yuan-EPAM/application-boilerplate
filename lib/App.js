"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _react = _interopRequireDefault(require("react"));

var _SignUp = _interopRequireDefault(require("./components/UI/SignUp"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function App() {
  return _react.default.createElement("div", null, _react.default.createElement(_SignUp.default, null));
}

var _default = App;
exports.default = _default;