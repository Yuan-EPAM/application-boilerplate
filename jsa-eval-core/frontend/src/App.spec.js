import React from 'react';
import renderer from 'react-test-renderer';
// import { mount } from 'enzyme';

import App from './App';

describe('test App ', () => {
  it('renders the App', () => {
    const appComponent = renderer.create(<App />);
    const tree = appComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
