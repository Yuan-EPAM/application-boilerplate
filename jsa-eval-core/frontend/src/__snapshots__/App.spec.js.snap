// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App  renders the App 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
